src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/ProPlanControls/BillingControls/BillingControls.tsx

```diff
@@ -0,0 +1,81 @@
+import { useEffect, useState } from 'react'
```

Comment:
Similar to the existing `BillingControls` component, but exclusively for the pro plan